ambry-store/src/main/java/com.github.ambry.store/HardDeleter.java

```diff
@@ -48,7 +48,7 @@
   public static final short Cleanup_Token_Version_V1 = 1;
   private static final String Cleanup_Token_Filename = "cleanuptoken";
   //how long to sleep if token does not advance.
-  static final long HARD_DELETE_SLEEP_TIME_ON_CAUGHT_UP = 10 * Time.MsPerSec;
+  static final long HARD_DELETE_SLEEP_TIME_MS_ON_CAUGHT_UP = 10 * Time.MsPerSec;
```

Comment:
Could you rename this to `HARD_DELETE_SLEEP_TIME_ON_CAUGHT_UP_MS`? Other than this LGTM.